### ISSUE TITLE
Fix HTML validation errors on the stats page

### ISSUE THREAD

**Issue by davep** [davep/blogmore#506]
If I [run the stats page through the HTML validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fblog.davep.org%2Fstats%2F) I get errors like this:

```
Error: The aria-label attribute must not be specified on any div element unless the element has a role value other than caption, code, deletion, emphasis, generic, insertion, paragraph, presentation, strong, subscript, or superscript.
From line 1, column 22983; to line 1, column 23088
rue></div><div aria-label="2026-01-01: 0 posts" class="streak-cell streak-cell-level-0" title="2026-01-01: 0 posts"></div>
```

This should be cleaned up; it seems the `aria-label` is unnecessary here.
